src/imitation/rewards/reward_nets.py

```diff
@@ -550,3 +575,200 @@ def __init__(
 
     def forward(self, state: th.Tensor) -> th.Tensor:
         return self._potential_net(state)
+
+
+class RewardEnsemble(RewardNetWithVariance):
+    """A mean ensemble of reward networks."""
+
+    members: nn.ModuleList
+
+    def __init__(
+        self,
+        observation_space: gym.Space,
+        action_space: gym.Space,
+        num_members: int = 5,
+        member_cls: Type[RewardNet] = BasicRewardNet,
+        member_kwargs: Mapping[str, Any] = {},
+        member_normalize_output_layer: Optional[Type[nn.Module]] = None,
```

Comment:
For now, I think `member_normalize_output_layer` isn't used even if it's specified because `forward_all()` and `forward()` don't used `predict_processed()` at all. I think we could directly remove `member_normalize_output_layer` unless we expect we'll ever have `RewardNetWrapper` as member_cls.